akka-docs/src/main/paradox/stream/operators/Source/queue.md

```diff
@@ -19,6 +19,21 @@ a buffer, if elements are pushed onto the queue faster than the source is consum
 a strategy specified by the user. Functionality for tracking when an element has been emitted is available through
 `SourceQueue.offer`.
 
+Using `Source.queue` you can push elements to the queue and they will be emitted to the stream if there is
+demand from downstream, otherwise they will be buffered until request for demand is received. Elements in the buffer
+will be discarded if downstream is terminated.
+
+In combination with the queue, the @ref[`throttle`](./../Source-or-Flow/throttle.md) operator can be used to control the processing to a given limit, e.g. `5 elements` per `3 seconds`.
```

Comment:
(I don't think `./` is needed?)

Reply:
Yeah, not needed, puts me at ease though (lol)